active/0014-rolling-cluster-upgrade.md

```diff
@@ -0,0 +1,122 @@
+# Rolling cluster upgrade
+
+## Change log
+
+* 2021-09-21: @k32 Initial draft
+* 2021-09-23: @k32 Applied remarks
+
+## Abstract
+
+Currently EMQ X upgrade procedure has two modes of upgrade:
+
+1. Patch releases.
+   Upgrading between the patch versions is done by patching the live nodes using Erlang hot code patching feature.
```

Comment:
What is this? do you have a ref to the document?  Is it a release zip file in the release dir?

Reply:
I am referring to this feature: https://docs.emqx.io/en/broker/v4.3/advanced/relup.html#hot-upgrade 
I will update the EIP to include the link